hack/yamls/knativeServiceCrd.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019 The Knative Authors
```

Comment:
That's a file that we need for testing, however it's located at `hack` directory. I suggest we always keep test related files under `test` directory.
Furthermore, I suggest a file that contains an third-party CRD also carries the CRD version on the name, or an easy way to identify the version.